paddle/fluid/framework/new_executor/interpreter/interpreter_util.cc

```diff
@@ -495,7 +509,7 @@ void BuildOpFuncList(const platform::Place& place,
         BuildVariableMap(output_name_map,
                          var_scope,
                          local_scope,
-                         /*allow_var_not_in_program=*/false,
+                         allow_var_not_in_program,
```

Comment:
Here `allow_var_not_in_program` should be `used_for_control_flow_op`

Reply:
Done